pyangbind/lib/xpathhelper.py

```diff
@@ -100,7 +107,7 @@ class FakeRoot(PybindBase):
   _pybind_generated_by = "FakeRoot"
 
   def __init__(self):
-    self._pyangbind_elements = {}
+    self._pyangbind_elements = OrderedDict()
```

Comment:
I'm not sure that order is important here, is there a reason that this needs to be ordered vs. simply being a dict?

Reply:
For consistency, if you are using YANG to generate native configuration it becomes super hard to test if the resulting text keeps changing because this is a dict and thus the order is not guaranteed.

Reply:
Testing is a good argument, happy for this to become ordered. AFAICS, it'll be ordered based on the order in the schema, which seems like what you would want.